recipe/meta.yaml

```
@@ -1,39 +1,47 @@
 {% set name = "dedupe" %}
-{% set version = "2.0.20" %}
-
+{% set version = "2.0.23" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dedupe-{{ version }}.tar.gz
-  sha256: 553e2f84f0c725c2bee154db990ff5460c65aca33eb58f4a64657e9dc41379b0
+  # switch to GH release archive to use tests
+  url: https://github.com/dedupeio/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d4f4e5bd777e3b8aeeac68eee5fbd36d4305cc8eac3115f778e3711a9dad8423
+  patches:
+    - patches/0001-fix-dependencies.patch
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [win]
+  # skip s390x: no btrees
+  skip: true  # [s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
+    - python
     - cython
     - pip
-    - python
+    - setuptools >=63
```

Lock setuptools https://github.com/dedupeio/dedupe/blob/f72d4a161bfc66c9e1de9b39e2bd7e01bcad3c49/pyproject.toml#L54C1-L55C1
```suggestion
    - setuptools ==63
```

---

@JamesRobertsonGames we prefer to keep the lower boundary instead of an exact pinning